starter-core/src/main/java/io/micronaut/starter/feature/aws/Cdk.java

```diff
@@ -164,6 +163,15 @@ private void populateDependencies() {
                 .groupId("org.junit.jupiter")
                 .artifactId("junit-jupiter-engine")
                 .test());
+        generatorContext.getFeatures().getFeature(AmazonApiGatewayHttp.class).ifPresent(feature -> {
```

Comment:
we could use 

```java
if (generatorContext.getFeatures().hasFeature(AmazonApiGatewayHttp.class)) {
```

since we are not going to use the unwrapped Optional.